src/components/Icon/index.examples.js

```diff
@@ -64,6 +64,13 @@ storiesOf('Icons')
     <Icon type='lesson' />
   ))
 
+  .addWithInfo('Refresh', () => (
+    <Icon
+      type='refresh' 
+      spin
+    />
+  ))
```

Comment:
@tayiorbeii  The previous way I had this file set up (and the other components) you didn't need these individual instances since `.addWithPropsCombinations` automatically took care of variations in props; but this works too - it just makes maintenance a bit more and the storybook has a lot more pages on it.